pkg/secrets-store/provider_client.go

```diff
@@ -263,18 +263,16 @@ func MountContent(ctx context.Context, client v1alpha1.CSIDriverProviderClient,
 		klog.InfoS("proto above 1MiB, secret sync may fail", "size", size)
 	}
 
-	if len(resp.GetFiles()) > 0 {
-		klog.V(5).Info("writing mount response files")
-		if err := fileutil.Validate(resp.GetFiles()); err != nil {
-			return nil, internalerrors.FileWriteError, err
-		}
-		if err := fileutil.WritePayloads(targetPath, resp.GetFiles()); err != nil {
-			return nil, internalerrors.FileWriteError, err
-		}
-	} else {
+	if len(resp.GetFiles()) == 0 {
 		// when no files are returned we assume that the plugin has not migrated
 		// grpc responses for writing files yet.
 		klog.V(5).Info("mount response has no files")
```

Comment:
Could we be more descriptive here? For end users who are seeing this info msg wouldn't make much sense unless they know the history of provider vs driver writing.
IMO it would be good to put a clear message stating what "no files" means.

Reply:
Updated the message and added a comment referencing the issue